dev/io.openliberty.microprofile.metrics.internal.5.0.monitor/bnd.bnd

```diff
@@ -0,0 +1,55 @@
+#*******************************************************************************
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+
+Bundle-Name: io.openliberty.microprofile.metrics.internal.5.0.monitor
+Bundle-SymbolicName: io.openliberty.microprofile.metrics.internal.5.0.monitor
+Bundle-Description: MicroProfile Metrics Monitor Bridge Auto Feature, version ${bVersion}
+
+bVersion=1.0
+
+javac.source: 11
+javac.target: 11
+
+Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=11))"
+Import-Package: \
+	jakarta.enterprise.*;version="[4.0,5)",\
+	org.eclipse.microprofile.metrics; version="[5.0.0,6.0.0)",\
+	jakarta.ws.rs.container;resolution:=optional,\
+	*
+
+Export-Package: io.openliberty.microprofile.metrics.internal.monitor
```

Comment:
Usually packages with internal in their name are not exported.  This explains why you had a separate package for those classes that were reference by other bundles.

Reply:
https://github.com/OpenLiberty/open-liberty/issues/22819